packages/celotool/src/lib/geth.ts

```diff
@@ -209,12 +231,12 @@ const validateGethRPC = async (
   handleError: HandleErrorCallback
 ) => {
   const transaction = await web3.eth.getTransaction(txHash)
-  if (!transaction.from || transaction.from.toLowerCase() !== from.toLowerCase()) {
-    handleError(!transaction.from || transaction.from.toLowerCase() !== from.toLowerCase(), {
-      location: '[GethRPC]',
-      error: `Expected "from" to equal ${from}, but found ${transaction.from}`,
-    })
-  }
+  const txFrom = transaction.from.toLowerCase()
+  const expectedFrom = from.toLowerCase()
+  handleError(!transaction.from || expectedFrom !== txFrom, {
```

Comment:
We have an `import { eqAddress } from '@celo/utils/lib/address'` that should avoid people having to lowerCase things